packages/manager/cypress/e2e/core/account/user-permissions.spec.ts

```diff
@@ -558,4 +559,246 @@ describe('User permission management', () => {
           .click();
       });
   });
+
+  it('can add full-access users with no redirect occurs', () => {
+    const mockUser = accountUserFactory.build({
+      username: randomLabel(),
+      restricted: false,
+    });
+
+    const username = randomLabel();
+    const newUser = accountUserFactory.build({
+      username: username,
+      email: `${username}@test.com`,
+      restricted: false,
+    });
+
+    const mockUserGrantsUpdated = grantsFactory.build();
+    const mockUserGrants = {
+      ...mockUserGrantsUpdated,
+      global: undefined,
+    };
+
+    mockGetUsers([mockUser]).as('getUsers');
+    mockGetUser(mockUser).as('getUser');
```

Comment:
```suggestion
    mockGetUser(mockUser);
```
It doesn't look like we're `wait`ing on this anywhere.